### PR TITLE
Updating process

### PR DESCRIPTION
**Pull request by glenrobson**
Linking in text from Tom's Ann Arbor document:

https://docs.google.com/document/d/1DEwDTU_PGV_1mXYJlAOtADxwfnqfov2gRO7NbMfl0t8/edit#

and discussions in the Cookbook group

### CODE REVIEW

**Comment by glenrobson** [comment]
Direct link: https://preview.iiif.io/cookbook/process_update/recipe/

File: recipe/index.md [hunk 1]
```diff
@@ -12,32 +12,54 @@ The [IIIF Presentation API][prezi3] specifies a standardised way to describe com
...
-* If approved, the pull request is merged, and the recipe becomes part of the published cookbook.
+* Clone the cookbook repository
+* Create a new branch locally to work on your recipe
+* The assignee will then create a draft pull request
+  * If you host the branch on the IIIF/cookbook repository the pull request build process will build a preview site. 
+* Once the assignee has completed their work on the pull request they will move it to a public pull request 
+* Review process
+  * Informal review by Cookbook group (announce on Cookbook slack channel)
+  * First review by Cookbook Editors group
+  * Final approval by the Technical Review Committee and after this the pull request will be merged to master)
```
Comment (cjcolvar) [inline recipe/index.md:37]: Does it make sense to explain what the process is like if the proposed recipe does not pass reviews at any of these stages?  Does it get informally reviewed again by the Cookbook group as the first step after receiving any feedback for changes and then proceed forward again?

**Comment by glenrobson** [comment]
(Automated comment): Branch available for testing at https://preview.iiif.io/cookbook/process_update

**Comment by glenrobson** [comment]
Made some quite drastic re-organization which I think makes it clearer:

The front page is now an intro and lists all of the recipes:

https://preview.iiif.io/cookbook/process_update/

This page is now exclusively about the process (removed links to recipes):

https://preview.iiif.io/cookbook/process_update/recipe/

All recipes now link to the front page and the auto generated list of recipes. See the following example:

https://preview.iiif.io/cookbook/process_update/recipe/0001-mvm-image/

**Pull request merged**